firefox-ios/Client/Frontend/UserContent/UserScripts/AllFrames/AtDocumentStart/LoginsHelper.js

```diff
@@ -444,6 +446,34 @@ window.__firefox__.includeOnce("LoginsHelper", function() {
     },
   }
 
+  const generatePassword = (rules) => {
+    let mapOfRules = null;
+
+    // This will be skipped for now, as we don't have the rules yet
+    // Once we have the rules, we can parse them and send an appropriate
+    // password to swift
+    if(rules) {
```

Comment:
Are these the same rules we added in the app?

Reply:
Yes it's the rules we added. This is an old comment. I will remove it. It should work now regardless if we have rules or not.

Reply:
Updated comment 😄